js/main.js

```diff
@@ -486,6 +489,120 @@ function getPokemonName() {
 }
 getPokemonName();
 
+$row.addEventListener('click', function (event) {
+  const selectedPokemon = event.target.getAttribute('src');
+  const selectedPokemonName = event.target.getAttribute('alt');
+
+  const existingPokemon = $oneRowPokemonPic.querySelector(`img[src="${selectedPokemon}"]`);
+
+  if (!existingPokemon) {
+    const $existingContent = $oneRowPokemonPic.querySelectorAll('.column-half');
+    for (let i = 0; i < $existingContent.length; i++) {
+      const element = $existingContent[i];
+      element.remove();
+    }
+
+    const $existingColumns = document.querySelectorAll('.column');
+    for (let i = 0; i < $existingColumns.length; i++) {
+      const element = $existingColumns[i];
+      element.remove();
+    }
+
+    const $pokemonPic = document.createElement('img');
+    $pokemonPic.setAttribute('src', selectedPokemon);
+    $pokemonPic.setAttribute('alt', selectedPokemonName);
+
+    const $weak = document.createElement('ul');
+    const $weak1 = document.createElement('li');
+    $weak1.textContent = 'No damage to:';
+    $weak.appendChild($weak1);
+
+    const $columnHalf = document.createElement('div');
+    $columnHalf.className = 'column-half';
+    $columnHalf.appendChild($weak);
+
+    const $columnSelect = document.createElement('div');
+    $columnSelect.className = 'column-half select';
+    $columnSelect.appendChild($pokemonPic);
+
+    const formattedPokemonName = selectedPokemon.replace('images/', '').replace('.gif', '');
+
+    const $pokemonTitle = document.createElement('h2');
+    $pokemonTitle.setAttribute('id', 'pokemon-name');
+    $pokemonTitle.textContent = formattedPokemonName.toUpperCase();
+
+    const $icon = document.createElement('i');
+    $icon.className = 'fa-solid fa-circle-plus fa-2xl';
+
+    $icon.addEventListener('click', function () {
+
+      const savedPokemon = {
+        name: formattedPokemonName.toUpperCase(),
+        image: selectedPokemon
+      };
+
+      let isPokemonSaved = false;
+      // eslint-disable-next-line no-undef
+      for (let i = 0; i < pokemonData.saved.length; i++) {
+        // eslint-disable-next-line no-undef
+        if (pokemonData.saved[i].name === savedPokemon.name) {
+          isPokemonSaved = true;
+          alert('Pokemon already saved');
+
+          break;
+        }
+      }
+      if (!isPokemonSaved) {
+        // eslint-disable-next-line no-undef
+        pokemonData.saved.unshift(savedPokemon);
+        location.reload();
```

Comment:
Forcing the page to reload is usually bad user experience, why are you forcing them to reload? If you're just trying to view swap, why not just view swap?